include/wabt/type-checker.h

```diff
@@ -77,7 +77,8 @@ class TypeChecker {
   Result EndBrTable();
   Result OnCall(const TypeVector& param_types, const TypeVector& result_types);
   Result OnCallIndirect(const TypeVector& param_types,
-                        const TypeVector& result_types);
+                        const TypeVector& result_types,
+                        const Limits& table_limits);
```

Comment:
does OnReturnCallIndirect need the same modification?

Reply:
Most likely.  I guess the test suite is lacking there?